lib/chef/win32/api/security.rb

```diff
@@ -415,6 +447,12 @@ def privilege(index)
         safe_attach_function :LookupPrivilegeNameW, [ :LPCWSTR, :PLUID, :LPWSTR, :LPDWORD ], :BOOL
         safe_attach_function :LookupPrivilegeDisplayNameW, [ :LPCWSTR, :LPCWSTR, :LPWSTR, :LPDWORD, :LPDWORD ], :BOOL
         safe_attach_function :LookupPrivilegeValueW, [ :LPCWSTR, :LPCWSTR, :PLUID ], :BOOL
+        safe_attach_function :LsaAddAccountRights, [ :pointer, :pointer, :pointer, :ULONG ], :NTSTATUS
```

Comment:
instead of `:pointer`, can we create aliases for the types in the documentation. https://github.com/chef/chef/blob/master/lib/chef/win32/api.rb#L49-L237